scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -209,7 +209,14 @@ def domain_cleanup(dom):
         dom.destroy()
 
     print("undefining {0}".format(dom.name()))
-    dom.undefineFlags(flags=libvirt.VIR_DOMAIN_UNDEFINE_NVRAM)
+    try:
+        dom.undefineFlags(flags=libvirt.VIR_DOMAIN_UNDEFINE_NVRAM)
+    except:
+        try:
+            dom.undefine()
+        except:
+            print("failed to undefine {0}".format(dom.name()))
+            raise
```

Comment:
mostly relevant when calling from vm_start where we do except libvirt.libvirtError to just print an informational message and continue.